api-inference-community/docker_images/fairseq/requirements.txt

```diff
@@ -0,0 +1,3 @@
+api-inference-community==0.0.19
+g2p_en
+git+git://github.com/pytorch/fairseq.git
```

Comment:
This might be a bit problematic. Could we add a specific commit here so we know what is deployed? If not, it gets hard to know what was in the deployed version which makes debugging harder

cc @Narsil

Reply:
This should only stay there for 1,2 weeks until fairseq has a new version.